.codeclimate.yml

```
@@ -0,0 +1,13 @@
+version: "2"         # required to adjust maintainability checks
+exclude_patterns:
+ - "**/node_modules/"
+ - "static/js/ckeditor/"
+ - "db/"
+ - "**/test/"
+checks:
+  similar-code:
+    config:
+      threshold: 160
+  identical-code:
+    config:
+      threshold: 55
```

Let's bump this up a bit higher, i think it's still too sensitive.
```suggestion
      threshold: 100
```